app/styles/_library/_objects.forms.scss

```diff
@@ -305,6 +305,19 @@ div.o-confirm-message p {
   background-color: rgb(var(--color-form-button));
 }
 
+.c-newsletter-form__button[disabled] {
+  cursor: not-allowed;
+  opacity: .6;
+}
+
+.c-newsletter-form .c-newsletter-form__button:hover[disabled] {
+  background-color: rgb(var(--color-newsletter-form-button));
```

Comment:
trying to get away from using a lot of organism/component-specific tokens as part of the token cleanup. It would save me some trouble if you could change this to use something more generic like `--color-button`. And If that gives you the wrong color for Gothamist, adding an override.

Reply:
👍